src/cargo/util/toml/mod.rs

```diff
@@ -655,6 +655,20 @@ impl TomlProfile {
             }
         }
 
+        if let Some(lto) = &self.lto {
+            if let StringOrBool::String(arg) = lto {
+                // try as I like I could not get `Deref` pattern matching to work here
```

Comment:
I would remove this as it is a "review" comment and not one to help others understand what is going on.